### PR TITLE
Add MQTT values - replaces https://github.com/Ardumower/Sunray/pull/106

### PR DESCRIPTION
**Pull request by hjw2015**
Included all comments and hints from #106

### CODE REVIEW

**Comment by disaster123** [comment]
there is no need to replace pull requests just force push the branch. Some notes added.

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 /config.h
+/alfred/build/**
```
Comment (disaster123) [inline .gitignore]: this is not mqtt related

File: alfred/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Sunray for Alfred
 
-## How to compile 'Sunray for Alred'
+## How to compile 'Sunray for Alfred'
```
Comment (disaster123) [inline alfred/README.md:3]: this is not mqtt related

File: sunray/Stats.h [hunk 3]
```diff
@@ -31,7 +31,6 @@ extern float statMowDistanceTraveled ; // meter
 extern float statTempMin;
 extern float statTempMax;
 
-
```
Comment (disaster123) [inline sunray/Stats.h:34]: why?

**Review by disaster123 — CHANGES_REQUESTED** [review]

File: sunray/comm.cpp [hunk 4]
```diff
@@ -1061,13 +1068,62 @@ void processWifiMqttClient()
...
+      MQTT_PUBLISH(gps.iTOW, "%lu", "/gps/tow")
+      
+      MQTT_PUBLISH(gps.lon, "%.8f", "/gps/lon")
+      MQTT_PUBLISH(gps.lat, "%.8f", "/gps/lat")
+      MQTT_PUBLISH(gps.height, "%.1f", "/gps/height")
+      MQTT_PUBLISH(gps.relPosN, "%.4f", "/gps/relNorth")
+      MQTT_PUBLISH(gps.relPosE, "%.4f", "/gps/relEast")
+      MQTT_PUBLISH(gps.relPosD, "%.2f", "/gps/relDist")
+      MQTT_PUBLISH((millis()-gps.dgpsAge)/1000.0, "%.2f","/gps/ageDGPS")
+      MQTT_PUBLISH(gps.accuracy, "%.2f", "/gps/accuray")
```
Comment (kraiz) [inline sunray/comm.cpp]: typo: accuray -> accuracy

**Review by kraiz — COMMENTED** [review]
I just spotted a typo.

Big thanks @hjw2015 !!

File: .gitignore (hunk 1 above)
Comment (hjw2015) [reply .gitignore]: Correct - more convinience that absolutely necessary.

File: alfred/README.md (hunk 2 above)
Comment (hjw2015) [reply alfred/README.md:3]: Correct - yet doing no harm

File: sunray/Stats.h (hunk 3 above)
Comment (hjw2015) [reply sunray/Stats.h:34]: because these two vairables are part of the provided MQTT values. BTW: these two lines were last edited by @greymfm

**Review by hjw2015 — COMMENTED** [review]
Looks good for me now - thanks to the reviewers ;-)

**Pull request merged**